bigchaindb/commands/bigchaindb.py

```diff
@@ -181,6 +181,28 @@ def run_upsert_validator_approve(args, bigchain):
         raise OperationError('Failed to vote for election')
 
 
+def run_upsert_validator_show(args, bigchain):
+    """Retrieves information about an upsert-validator election
+
+    :param args: dict
+        args = {
+        'election_id': the transaction_id for an election (str)
+        }
+    :param bigchain: an instance of BigchainDB
+    :return: (public_key, power, node_id) for the proposed change to the validator set
+    """
+
+    election = bigchain.get_transaction(args.election_id)
+
+    new_validator = election.asset['data']
+
+    public_key = new_validator['public_key']
+    power = new_validator['power']
+    node_id = new_validator['node_id']
+
+    return f'public_key={public_key}\npower={power}\nnode_id={node_id}'
```

Comment:
I had a discussion with @ldmberman, in that we decided that we can definitely state whether the election has concluded or is ongoing or cannot be concluded i.e. the following statuses for a given election should be listed,
- `ongoing`: Implies that the election is in progress and that a supermajority vote has been received yet.
- `concluded`: Implies that the election was concluded and the validator change was applied.
- `inconclusive`: implies that the election received a supermajority vote but couldn't conclude because the validator set at the time of conclusion was different from that when the election was proposed.

So, in view of the above following would be the output of the command,
```bash
$ bigchaindb upsert-validator show ELECTION_ID
public_key=<e_pub_key>
power=<e_power>
node_id=<e_node_id>
status=ongoing
```